blacklight-cornell/app/controllers/search_controller.rb

```diff
@@ -1,86 +1,93 @@
 require "cgi"
 
 class SearchController < ApplicationController
+  include LoggingHelper
+
   before_action :heading
+
   def heading
-   @heading='Search'
+    @heading = "Search"
```

Comment:
`'` vs `"` - does the team have a preference? I've been using `'`, except for when it's necessary to use `"` (e.g. string interpolation)! Maybe we can bring up in a standup with the rest of the team to get consensus?

Reply:
This is something my editor does as part of "whitespace" cleanup. I haven't bothered to find the setting to turn it off, but it's not a particular preference of mine.